authors/apps/comments/tests/test_comments_endpoints.py

```diff
@@ -240,3 +243,47 @@ def test_cannot_delete_reply_if_you_didnt_author_it(self):
             "You can only delete a reply you authored",
             response.data['response'])
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
```

Comment:
Nice work on the tests but I have not seen tests that are passing for when you can comment on text

Reply:
Alright, I'll add those.

Reply:
Done.